src/Forms/Prism.Forms/Navigation/NavigationException.cs

```diff
@@ -29,4 +30,14 @@ public NavigationException(string message, Page page, Exception innerException)
 
         public Page Page { get; }
     }
+
+    public class AmbiguousPathException : Exception
+    {
+        public string Name { get; }
+
+        public AmbiguousPathException(string name)
+        {
+            Name = name;
+        }
+    }
```

Comment:
this should be in it's own file... see notes about using it as the inner exception to the NavigationException

Reply:
I don't think that this is needed since Prism will just navigate to the first matching Tab and continue navigating within that tab.

In [your update](https://brianlagunas.com/whats-new-in-prism-for-xamarin-forms-7-0/) for Prism 7.0 you wrote, that having multiple NavigationPages is not an issue anymore since we are using the selectedTab parameter.